docs/book/src/developer/providers/migrations/v1.9-to-v1.10.md

```diff
@@ -17,4 +17,10 @@ maintainers of providers and consumers of our Go API.
 
 ### Other
 
+- The following functions for `E2EConfig` in `sigs.k8s.io/cluster-api/test/framework/clusterctl` got renamed ([#11743](https://github.com/kubernetes-sigs/cluster-api/pull/11743)):
```

Comment:
I think we should only describe the delta between v1.9 and v1.10

I think GetVariableBestEffort didn't exist on 1.9? (I might be wrong)